src/components/ConnectWallet/Wallets/TrezorSuite.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <connection-details name="Trezor Suite" requires="electrum">
+    <step-list>
+      <step>
+        Open Trezor Suite and connect your Trezor.
+      </step>
+      <step>
+        Make sure Tor is enabled in Trezor Suite.
+      </step>
```

Comment:
Maybe we should make this step clearer, like this?
```suggestion
      <step>
        Go to <span class="font-weight-bold">Settings > Application</span> and make sure <span class="font-weight-bold">"Tor"</span> is enabled.
      </step>
```